src/App.php

```diff
@@ -238,14 +238,14 @@ static function (int $severity, string $msg, string $file, int $line): bool {
     }
 
     /**
-     * Register a modal view.
-     * Fomantic-ui Modal are teleported in HTML template
+     * Register a portal view.
+     * Fomantic-ui Modal or atk Panel are teleported in HTML template
      * within specific location. This will keep track
-     * of modals when terminating app using json.
+     * of them when terminating app using json.
      */
-    public function registerModal(Modal $modal): void
+    public function registerPortals(View $portal): void
```

Comment:
I would prefer `Modal|Panel|Right` phpdoc here to prevent others to use this wrongly.

Reply:
done!